fineract-core/src/main/java/org/apache/fineract/infrastructure/core/service/DateUtils.java

```diff
@@ -78,6 +79,14 @@ public static LocalDateTime getLocalDateTimeOfSystem() {
         return LocalDateTime.now(ZoneId.systemDefault()).truncatedTo(ChronoUnit.SECONDS);
     }
 
+    public static LocalDateTime getAuditLocalDateTime() {
```

Comment:
This change belongs to this PR as I already use it for the datatables.
I suggested to create this method and use it all over the application instead of the system time, even if it does not cause any problem for now, because of the current server settings.
This should be done by a separate request, separate story, separate PR. For now I introduced these methods and will use for datatables and savings related entities. Savings changes will come with the next PR.